src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/aggregation/aggregation.component.scss

```diff
@@ -100,3 +100,11 @@
     top: -19px;
   }
 }
+
+.aggregation.disabled {
+  mat-panel-title {
+    &.mat-expansion-panel-header-title {
+      opacity: 0.7;
```

Comment:
shouldn't the opacity be part of the theme.scss?

Reply:
Fixed